// @antfu/ni/dist/index.cjs
'use strict';

const index = require('./shared/ni.c76609cc.cjs');
require('node:fs');
require('node:path');
require('node:process');
require('node:module');
require('child_process');
require('path');
require('process');
require('stream');
require('readline');
require('os');
require('tty');
require('events');
require('node:os');
require('fs');
require('fs/promises');
require('node:fs/promises');



exports.AGENTS = index.AGENTS;
exports.CLI_TEMP_DIR = index.CLI_TEMP_DIR;
exports.COMMANDS = index.COMMANDS;
exports.INSTALL_PAGE = index.INSTALL_PAGE;
exports.LOCKS = index.LOCKS;
exports.UnsupportedCommand = index.UnsupportedCommand;
exports.cmdExists = index.cmdExists;
exports.constructCommand = index.constructCommand;
exports.detect = index.detect;
exports.exclude = index.exclude;
exports.formatPackageWithUrl = index.formatPackageWithUrl;
exports.getCliCommand = index.getCliCommand;
exports.getCommand = index.getCommand;
exports.getConfig = index.getConfig;
exports.getDefaultAgent = index.getDefaultAgent;
exports.getGlobalAgent = index.getGlobalAgent;
exports.limitText = index.limitText;
exports.parseNa = index.parseNa;
exports.parseNi = index.parseNi;
exports.parseNlx = index.parseNlx;
exports.parseNr = index.parseNr;
exports.parseNu = index.parseNu;
exports.parseNun = index.parseNun;
exports.remove = index.remove;
exports.resolveCommand = index.resolveCommand;
exports.run = index.run;
exports.runCli = index.runCli;
exports.serializeCommand = index.serializeCommand;
exports.writeFileSafe = index.writeFileSafe;
